### PR TITLE
Updated LGP and LMP versions in Sample projects used in UI tests

### LINKED ISSUE #986: Update the build files in the sample projects used in UI test cases to include LGP version 3.8.1 or later and LMP version 3.10.1 or later.
Update the LGP and LMP versions in the sample projects in UI test cases.

### PR DESCRIPTION
**Pull request by anusreelakshmi934**
Fixes #986

### CODE REVIEW

**Review by aparnamichael — APPROVED** [review]

File: src/test/resources/projects/gradle/singleModGradleMP/build.gradle [hunk 1]
```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'war'
-    id 'io.openliberty.tools.gradle.Liberty' version '3.6.2'
+    id 'io.openliberty.tools.gradle.Liberty' version '3.8.3'
```
Comment (turkeylurkey) [inline src/test/resources/projects/gradle/singleModGradleMP/build.gradle:3]: The problem should also exist in `sampleGradleMPLSApp`.

File: src/test/resources/projects/maven/singleModMavenMP/pom.xml [hunk 2]
```diff
@@ -77,7 +77,7 @@
             <plugin>
                 <groupId>io.openliberty.tools</groupId>
                 <artifactId>liberty-maven-plugin</artifactId>
-                <version>3.8.2</version>
+                <version>3.10.3</version>
```
Comment (turkeylurkey) [inline src/test/resources/projects/maven/singleModMavenMP/pom.xml:80]: In some test cases `liberty-maven-plugin` is version 3.7.1. We should think about using the same version for all the tests.

File: src/test/resources/projects/gradle/singleModGradleMP/build.gradle (hunk 1 above)
Comment (anusreelakshmi934) [reply src/test/resources/projects/gradle/singleModGradleMP/build.gradle:3]: Changed `sampleGradleMPLSApp` also.

File: src/test/resources/projects/maven/singleModMavenMP/pom.xml (hunk 2 above)
Comment (anusreelakshmi934) [reply src/test/resources/projects/maven/singleModMavenMP/pom.xml:80]: @turkeylurkey So you mean to change all `liberty-maven-plugin` to the latest? `3.7.1` version is present in src/test/resources/files/smNLTRestProject/maven/pom.xml ivdeya and src/test/resources/projects/maven/jakarta-sample/pom.xml.

Comment (anusreelakshmi934) [reply src/test/resources/projects/maven/singleModMavenMP/pom.xml:80]: Similarly with the gradle version in src/test/resources/files/smNLTRestProject/gradle/build.gradle. also?

Comment (turkeylurkey) [reply src/test/resources/projects/maven/singleModMavenMP/pom.xml:80]: Yes

**Review by turkeylurkey — APPROVED** [review]

**Review by aparnamichael — APPROVED** [review]

**Review by TrevCraw — APPROVED** [review]

**Pull request merged**